docs/src/aleph_bft_api.md

```diff
@@ -142,7 +152,7 @@ def get_data():
 We simply fetch at most 100 transactions from the local pool and return such a list of transactions.
 
 ```
-def send_ordered_batch(batch):
+def data_finalized(batch):
```

Comment:
Not batch, but a single piece of data. This also requires changes in the code below and some explanation as to when a block should be created at all – no natural batches in this case; probably collect transactions until some timeout or limit is reached?

Reply:
Since the `Data here is a `Vec` of txs, Im just using it as is (So without collecting and timeout). Let me know if this is fine by you.